cek/builtins.go

```diff
@@ -3150,7 +3266,11 @@ func replicateByte[T syn.Eval](m *Machine[T], b *Builtin[T]) (Value[T], error) {
 	if !size.IsInt64() {
 		return nil, errors.New("replicateByte: size too large")
 	}
-	sizeInt := int(size.Int64())
+	sizeInt64 := size.Int64()
+	if sizeInt64 > int64(math.MaxInt) {
+		return nil, errors.New("replicateByte: size too large")
+	}
+	sizeInt := int(sizeInt64)
```

Comment:
The value of `math.MaxInt` and the underlying type for `int` will differ for 32-bit vs. 64-bit platforms. We should probably use a fixed-size type instead